tests/PromiseTests.fs

```diff
@@ -322,3 +322,21 @@ describe "Promise tests" <| fun _ ->
         p.``then``(fun _ -> promiseExecutionCount |> equal 1)
         p.``then``(fun _ -> promiseExecutionCount |> equal 1)
         p.``then``(fun _ -> promiseExecutionCount |> equal 1)

```

Comment:
Question about the test framework, does it handle correctly that it might execute asynchronously and so equal will be called potentially later ?